lib/components/chat_history/twitch/subscription_event.dart

```diff
@@ -72,6 +72,16 @@ class TwitchSubscriptionMessageEventWidget extends StatelessWidget {
   const TwitchSubscriptionMessageEventWidget(this.model, {Key? key})
       : super(key: key);
 
+  Color chipBackgroundColor(String tier, BuildContext context) {
+    if (tier == '2000') {
+      return  const Color(0xFFFFEA00);
+    } else if (tier == '3000') {
+      return  const Color(0xFFD2D2e6);
```

Comment:
```suggestion
      return const Color(0xFFFFEA00);
```
Tier 3 is more commonly subscribed to lets make it a distinguished color.